watertap/unit_models/tests/test_ion_exchange_0D.py

```diff
@@ -930,13 +930,13 @@ def obj_rule(m):
         results = solver.solve(m, tee=False)
         assert_optimal_termination(results)
 
-        assert pytest.approx(440372.278, rel=1e-5) == value(
+        assert pytest.approx(453807.598, rel=1e-5) == value(
```

Comment:
Why are the ion exchange costs changing? Was a bug fixed?

Reply:
Yes, look at f3473c3. There were some uncaught issue with units.